discord/bot.py

```diff
@@ -812,9 +812,11 @@ async def process_application_commands(self, interaction: Interaction, auto_sync
             command = self._application_commands[interaction.data["id"]]
         except KeyError:
             for cmd in self.application_commands:
+                guild_id = interaction.data.get("guild_id")
+                if guild_id: guild_id = int(guild_id)
```

Comment:
```suggestion
                if guild_id: 
                    guild_id = int(guild_id)
```
We should use indentation even for simple assignments as it's easier to read and conforms to the rest of the code style in the library.